vocabularies.md

```diff
@@ -0,0 +1,283 @@
+          },
+          "self": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/srsc"
+          }
+        }
+      },
+      {
+        "id": "common_types",
+        "name": "common_types",
+        "scrollable": true,
+        "hierarchical": false,
+        "type": "vocabulary",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/srsc/entries"
+          },
+          "self": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/srsc"
+          }
+        }
+      },
+      {
+        "id": "common_iso_languages",
+        "name": "common_iso_languages",
+        "scrollable": true,
+        "hierarchical": false,
+        "type": "vocabulary",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_iso_languages/entries"
+          },
+          "self": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_iso_languages"
+          }
+        }
+      }
+    ]
+  },
+  "_links": {
+    "self": {
+      "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies"
+    }
+  },
+  "page": {
+    "size": 20,
+    "totalElements": 3,
+    "totalPages": 1,
+    "number": 0
+  }
+}
+```
+
+## Single Vocabulary
+**/api/submission/vocabularies/<:vocabulary-name>**
+
+Provide detailed information about a specific controlled vocabulary. The JSON response document is as follow
+```json
+{
+  "id": "srsc",
+  "name": "srsc",
+  "scrollable": false,
+  "hierarchical": true,
+  "preloadLevel": 1,
+  "type": "vocabulary"
+}
+```
+
+Attributes
+* id: the id of the vocabulary, it is the same than the name
+* name: see id
+* scrollable: if true mean that it is possible to scroll all the entries in the authority without providing a filter parameter, see (vocabulary entries)[vocabularies.md#vocabulary-entries]
+* hierarchical: if true means that the vocabulary expose a tree structure where some entries are parent of others
+* preloadLevel: for hierarchical vocabularies express the preference to preload the tree at a specific level of depth (0 only the top nodes are shown, 1 also their children are preloaded and so on)
+
+Exposed links:
+* entries: get entries from the controlled vocabulary
+
+## Linked entities
+### controlled vocabulary entries
+**/api/submission/vocabularies/<:vocabulary-name>/entries**
+
+It returns the entries in the vocabulary in response to either a user input or a scrollable list, see below 
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* metadata: the metadata field name (e.g. "dc.type") for which the controlled vocabulary is used: mandatory. The entries returned by a vocabulary may be different for different metadata fields.
```

Comment:
When can they be different for separate metadata fields but still using the same vocabulary ID? Can you provide a sample?
Otherwise I'd recommend to remove this parameter

Reply:
this is used by the SOLRAuthority https://github.com/DSpace/DSpace/blob/30ad7676ef92df507cffb7a2040f5ef3ee5e56b9/dspace-api/src/main/java/org/dspace/content/authority/SolrAuthority.java#L79 and related to the previous discussion with Tim see https://github.com/DSpace/Rest7Contract/pull/128#discussion_r437661208

Reply:
Since this seems to be confusing to Tim and me, I'd suggest to add an example to the contract. This doesn't have to be a full JSON example, but simply state that for the vocabulary-name `solr` (or what the actual name is), the behavior differs based on the metadata and collection because it's a single vocabulary whose data is retrieved via the dspace.cfg `choices.plugin.dc.contributor.author = SolrAuthorAuthority` (if that's how this field is determined)

Reply:
So if I understand it correctly, there are already separate endpoints per solr field. The fact that this vocabulary is named `SolrAuthorAuthority` does determine which entries to return (and the collection or metadata field still won't impact it)?

Reply:
not exactly. It is named SolrAuthorAuthority because this is the name used in our dspace.cfg examples. It can be configured over more than one metadata and as you can see in the above link to the existing sourcecode (unchanged from previous version) the SolrAuthority will use the metadata information to query the authority core

Reply:
I just added the [below comment](https://github.com/DSpace/Rest7Contract/pull/128#discussion_r445855817) which applies here as well.  `metadata` needs to be a required field because of value-pairs (which require it).  So, I think we'll have to keep this as-is for now in the contract.  If need be, we can always merge this as-is & I will gladly add more text to describe *why* both collection & metadata are required at this time -- the best example of both is value-pairs.

Reply:
I'd recommend to add in at least some detail on this line explaining this is currently only applicable for the solr authority, since it's still unclear from this contract.
Value pairs don't differ per metadata field, only per vocabulary-name
That should be a really small change, and this is very obviously an unclear area